docs/release-notes.txt

```diff
@@ -8,6 +8,31 @@
  Version 0.17
 **************
 
+Version 0.17.15
+===============
+
+**Release Date:** April 22, 2022
+
+**Breaking Changes**
+
+-  API: Endpoints for getting or updating a user now accept a ``userId`` instead of ``username`` as
+   the path parameter.
+
+**Bug Fixes**
+
+-  Fix an issue where deleted experiments would get stuck in a ``DELETING`` state indefinitely due
+   to their checkpoint GC tasks not to completing.
+
+-  API: Fix an issue where a reported job state could be stale due to a faulty caching mechanism.
+   This could have resulted in an experiment showing in `queued` or `scheduled` state, either in CLI
```

Comment:
Change to "... state in the CLI or WebUI when it was in the other state."
Or, should it really be "... in another state."?

Reply:
I think the original wording is correct but I'll have @hamidzr confirm.

Reply:
I'm using "other" to communicate that it could have been only between two options eg if it wasn't `scheduled` it should have been `queued` and not just any other state.
"another" seems correct as well, but I think it would be less accurate.

Reply:
I agree with leaving it as is in that case, although, it's not clear that those are the only possible states; if you think most users should know that, it's fine.